shared/editor/lib/code.ts

```diff
@@ -0,0 +1,65 @@
+import Storage from "../../utils/Storage";
+
+const RecentStorageKey = "rme-code-language";
```

Comment:
```suggestion
const LegacyStorageKey = "rme-code-language";
```

I think it would be preferable to explicitly mark this as legacy, just use it to fill out the new key on instantiation and delete. `getRecentCodeLanguage` can then just use the first item in frequent code languages going forward?

Reply:
I'm afraid the frequent code languages is an object of language to its frequencies - it's not possible to find the recent (last used) language from them.
The idea of maintaining a frequency object is to show the user their most probable preferred languages based on their usage history, rather than just the recent (say last 5) selections.

Do you think we should simplify this and maintain just the recent langs in an array?

Reply:
Gotcha, so the most used might not be the most recent – okay that seems fine